src/test/scala/com.snowplowanalytics.dataflow/SimpleSpec.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+import org.specs2._
+import com.snowplowanalytics.dataflow.streaming.SimpleEvent
+
+class SimpleSpec extends Specification { def is = s2"""
+
+	Spec to test JSON parsing
+	
+	Example JSON event string:
+	
+	'{	
+  		"timestamp": "2015-06-05T12:54:43.064528",
+  		"type": "Green",
+  		"id": "4ec80fb1-0963-4e35-8f54-ce760499d974"
+	}'
+
+	
+	The previous JSON string should
+		have a `type` field equal to 'Green'					$e1
+																"""
+
+	val json_event = "{\"timestamp\": \"2015-06-05T12:54:43.064528\",\"type\": \"Green\",\"id\": \"4ec80fb1-0963-4e35-8f54-ce760499d974\"}"
+	val se = SimpleEvent.fromJson(json_event)
+	def e1 = if (se.isDefined) {
+		se.get.`type` must beEqualTo("Green")
+	} else {
+		 se must beEqualTo(None)
+	}
+}
```

Comment:
indent + try to test both failing and succeeding cases.

Also are there any test utilities for dataflow? It'd be great to provided tests for `bucketEvents` for example.
I think that would bring a lot of value to this example project.